content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md

```diff
@@ -0,0 +1,96 @@
+2. [Configure a Thing sketch](#create-thing)
+3. [Upload the sketch to the device](#upload-sketch)
+
+---
+
+## Set-up instructions
+
+<a id="add-the-device"></a>
+
+### Step 1: Add the device
+
+1. Open Arduino IoT Cloud and open the [Devices tab](https://create.arduino.cc/iot/devices).
+
+2. Click the **Add Device** button in the top-right or middle of the page.
+
+   ![The two location that the "Add Device" button can appear.](img/device-add-button.png)
+
+3. Select your device type:
+
+   * _Set up an Arduino device_: For Arduino boards.
+   * _Set up a 3rd Party Device_: ESP8266, ESP32, and more.
+
+   > Hover over the <kbd>i</kbd> for a full list of compatible devices.
+
+4. **Proceed with the instructions to add your device.**
+
+   * _ESP32 and ESP8266_: Make sure to save any Device ID and Secret key that's provided.
+   * _LoRaWAN<sup>®</sup>:_ Make sure you select the [right frequency band for your region](https://www.thethingsnetwork.org/docs/lorawan/frequencies-by-country/). When completing the setup you'll receive a registration email for your [The Things Stack Network Account](https://docs.arduino.cc/cloud/iot-cloud/tutorials/cloud-lora-getting-started#step-2-registering-the-things-stack-network-account).
+
+5. When finished, your new device will be added to your list of devices.
+
+   ![A MKR WiFi 1010 in the Device tab.](img/iot-device-added.png)
+
+<a id="create-thing"></a>
+
+### Step 2: Create and configure a Thing
+
+<!-- The network parameters will be written to the `thingProperties.h` and `Secret` sketch files. But a sketch is only generated when a variable is added, so we'll do that first. -->
+
+1. Open the [Things tab](https://create.arduino.cc/iot/things).
+
+2. Click **Create Thing**.
+
+   ![The two location that the "Create Thing" button can appear.](img/iot-thing-create.png)
+
+3. Configure **Variables**, **Device**, and **Network** for the new Thing:
+
+   ![A new, unconfigured Thing.](img/thing-setup-before.png)
+
+   1. Click the **Device selection** button and select your device.
+
+   2. Click the **Network configure** button and enter the network credentials.
+
+      * _Wi-Fi:_ Enter your Wi-Fi name (SSID) and password (encryption key).
+      * _Arduino SIM_: [Use these credentials](https://support.arduino.cc/hc/en-us/articles/360013825159-What-are-the-credentials-for-the-Arduino-SIM-card).
+      _Arduino with LoRa<sup>®</sup>:_ The credentials are added automatically.
+      * _ESP32 and ESP8266_: Enter the secret key that was provided when adding the device.
+
+   3. Click **Add variable** and add one variable with any settings.
+
+   ![A Thing with an asociated device, network configuration, and added variable.](img/thing-setup-after.png)
+
+4. Your sketch will now be generated and can be viewed in the Sketch tab.
+
+<a id="upload-sketch"></a>
+
+### Step 3: Upload the Thing sketch
+
+1. With the Thing open, open the **Sketch** tab.
+
+2. Click the **Upload button** and wait for the process to complete.
```

Comment:
Maybe a step should be added here about installing or starting Arduino Create Agent if it is not already running?

Reply:
Applies to "Add device" as well, added to introduction in 8dd0e25